docs/installation.rst

```diff
@@ -4,97 +4,99 @@
 Installation
 ************
 
-Requirements
-============
+The SunPy project `maintains a range of packages <https://sunpy.org/project/affiliated>`__ that leverage the wider ecosystem of scientific Python packages for solar physics.
 
-sunpy requires Python 3.8 or higher.
+Python distribution
+===================
+These instructions will set you up with `miniforge <https://conda-forge.org/docs/user/introduction.html>`__, which makes it easy to install and manage Python packages.
 
-Installing Scientific Python and sunpy
-======================================
+`To install the miniforge Python distribution follow these instructions <https://github.com/conda-forge/miniforge#install>`__.
 
-sunpy is part of the wider ecosystem of scientific Python packages for solar physics.
-Therefore a working sunpy installation is more about installing the scientific Python ecosystem than sunpy itself.
+Installing miniforge will set the default conda channel to ``conda-forge``, which is required for installing ``sunpy`` and many other packages in the scientific Python ecosystem.
```

Comment:
I think the concept of a "channel" needs to be introduced before this sentence to avoid confusion.

Reply:
How about I delete this line altogether? The conda-forge docs explain this and I rather not repeat that in a worse manner.

Reply:
👍